html_app/components/FACSModelFactory.js

```diff
@@ -64,7 +63,7 @@ scb.components.FACSModelFactory = function scb_components_FACSModelFactory(model
              */
         }
         self.shape_to_data = function (state) {
-            var shape = state.shape;
+            var shape = ('' + state.shape).toLowerCase();
```

Comment:
That's right. I moved this transformation from the if statement to the declaration.